torch/csrc/utils/six.h

```diff
@@ -2,6 +2,7 @@
 
 #include <pybind11/pybind11.h>
 #include "torch/csrc/utils/structseq.h"
+#include "torch/csrc/utils/object_ptr.h"
```

Comment:
super nit: Please change quotes to brackets here and line above.

Reply:
@VitalyFedyunin Sure! changed.